.github/workflows/trivy.yml

```diff
@@ -25,7 +23,7 @@ jobs:
       - name: Run Trivy vulnerability scanner
         uses: aquasecurity/trivy-action@2a2157eb22c08c9a1fac99263430307b8d1bc7a2
         with:
-          image-ref: ghcr.io/ondat/trousseau:${{ github.sha }}'
+          image-ref: ghcr.io/ondat/trousseau:latest'
```

Comment:
Could you please describe why `latest` is the best to use here?

Reply:
I modified the trivy.yml for ondemand only as there is a dedicated definition for the e2e and I pointed to latest for that purpose. 
Would you put instead?

Reply:
since we have the trivy vulnerability scanning done for every PR, we can potentially remove this ondemand scan